templates/matchingDropDown.jsx

```diff
@@ -205,13 +209,14 @@ export default function MatchingDropDown(props) {
           left: buttonOffsetLeft,
           width: buttonWidth
         }}
+        id={`${_id}-matching-item-${_itemIndex}__list`}
         role="listbox"
         tabIndex="-1"
         onBlur={onListBlur}
         ref={list}
         disabled={!_isEnabled}
+        aria-labelledby={props.questionTitleId}
```

Comment:
```suggestion
        aria-labelledby={props.ariaLabelledBy}
```
This one also needs updating?